habitat_baselines/common/rollout_storage.py

```diff
@@ -84,9 +88,11 @@ def __init__(
 
         assert (self._num_envs % self._nbuffers) == 0
 
-        self.numsteps = numsteps
+        self.num_steps = numsteps
         self.current_rollout_step_idxs = [0 for _ in range(self._nbuffers)]
 
+        self.device = torch.device("cpu")
```

Comment:
Add a comment explaining why the device needs to be the cpu

Reply:
It doesn't need to be the CPU, you can move the rollout storage to another device with the `.to` method. The default device for a torch tensor is the CPU, so everything we've allocated is on the CPU. This is just tracking that fact.